.circleci/config.yml

```
@@ -5,6 +5,16 @@ orbs:
   android: circleci/android@2.0.3
   node: circleci/node@4.7.0
 
+commands:
+  # Install latest CMake version, needed for building Hermes for visionOS which requires at least CMake 3.29. 
+  brew-install-cmake:
+    steps:
+      - run:
+          name: Install CMake
+          command: |
+            brew update
+            brew install cmake
```

Can we pin the CMake version here?
```suggestion
            brew install cmake@3.29.0
```

---

It needs to be 3.29.2, I'll update this shortly

---

Updated!

---

It looks like it doesn't work :( `brew install cmake@3.29.2` fails..

From Homebrew docs: 

> Homebrew is a [rolling release](https://en.wikipedia.org/wiki/Rolling_release) package manager so it does not support installing arbitrary older versions of software. If your software needs specific pinned versions, consider [whalebrew](https://github.com/whalebrew/whalebrew) lines in your Brewfile to install [Docker](https://www.docker.com/) containers.